<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File System</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rsc.qrc">
    <normaloff>:/icons/icons/2.png</normaloff>:/icons/icons/2.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>TreeView</string>
    </property>
    <widget class="QTreeView" name="treeView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>451</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>441</width>
       <height>101</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QGroupBox" name="groupBox_filter">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>431</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QCheckBox" name="checkBox_filter">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>20</y>
         <width>73</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>文件过滤</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>181</width>
         <height>24</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>*.cpp;*.h</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>*.*</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>*.ui</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>*.jpg;*.png</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>*.txt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>*.exe</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_apply">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>20</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>应用</string>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>401</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QRadioButton" name="radioButton_only_dir">
       <property name="text">
        <string>只显示目录</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_all">
       <property name="text">
        <string>显示目录和文件</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>371</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>ListView</string>
    </property>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>351</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>371</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>TableView</string>
    </property>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>351</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_filepath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>731</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>路径：</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>851</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_filename">
     <property name="text">
      <string>文件名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_filesize">
     <property name="text">
      <string>文件大小：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_filetype">
     <property name="text">
      <string>节点类型：</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_is_dir">
     <property name="text">
      <string>是否为目录</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_root_dir"/>
   <addaction name="separator"/>
   <addaction name="action_quit"/>
  </widget>
  <action name="action_root_dir">
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/icons/icons/3.png</normaloff>:/icons/icons/3.png</iconset>
   </property>
   <property name="text">
    <string>设置根目录</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/icons/icons/5.png</normaloff>:/icons/icons/5.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="rsc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
